tests/php/CRF_CSS_Optimization_Test.php

```
@@ -0,0 +1,481 @@
+    public function tearDown(): void {
+        parent::tearDown();
+        $this->clear_all_css_cache();
+    }
+    
+    /**
+     * Clear all CSS cache entries
+     */
+    private function clear_all_css_cache() {
+        global $wp_transients;
+        if (isset($wp_transients)) {
+            foreach ($wp_transients as $key => $value) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$value'. (Unused Code Rules)

(UnusedLocalVariable)